src/context/playlist-context.js

```diff
@@ -1,38 +1,38 @@
-import {
-    createContext,
-    useContext,
-    useState,
-    useEffect,
-  } from "react";
-import { useLocalStorageGetItem } from "../hooks/customHooks";
-  import { getPlaylists } from "../serverCalls/likeCalls";
-  const PlaylistsContext = createContext();
-  function PlaylistsProvider({ children }) {
-    const [Playlists, setPlaylists] = useState([]);
-    const userToken=useLocalStorageGetItem("shed-video-token");
+// import {
+//     createContext,
+//     useContext,
+//     useState,
+//     useEffect,
+//   } from "react";
+// import { useLocalStorageGetItem } from "../hooks/customHooks";
+//   import { getPlaylists } from "../serverCalls/likeCalls";
+//   const PlaylistsContext = createContext();
+//   function PlaylistsProvider({ children }) {
+//     const [Playlists, setPlaylists] = useState([]);
+//     const userToken=useLocalStorageGetItem("shed-video-token");
```

Comment:
Please remove commented code if it is not being used.

Reply:
ok